### DISCUSSION TITLE
Problems with opening dpgen run again after a task interruption

### DISCUSSION THREAD

**Discussion by SuperAiSks** [deepmodeling/dpgen#1356] [Q&A]
While working on 01.model_devi, a task was interrupted for some reason, leaving none of the later exploratory tasks calculated. How should I do the exploration again？
What about resubmitting dpgen run param.json machine.json? Do i need to modify the record file?

**Comment by njzjz** [comment]
You can rerun dpgen without modifying the record file.

**Comment by SuperAiSks** [comment]
Ok, thanks! 
Does dpgen automatically recognize completed exploratory tasks from uncompleted exploratory tasks? Does the jr.json file in the 01.model_devi directory need to be modified?
